packages/edit-site/src/components/editor/index.js

```diff
@@ -265,6 +264,15 @@ export default function Editor( { isLoading, onClick } ) {
 
 	return (
 		<>
+			<GlobalStylesRenderer />
+			<EditorKeyboardShortcutsRegister />
+			{ isEditMode && <BlockKeyboardShortcuts /> }
+			{ showVisualEditor && (
+				<>
+					<TemplatePartConverter />
+					<PatternModal />
+				</>
+			) }
```

Comment:
Most of these don't need to be in the "content" area. So I just moved them to align the content area with the post editor.